<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMessage</class>
 <widget class="QDialog" name="MyMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="m_editText">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Message:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnInfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnWarn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Warn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnError">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Error</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
